docs/api/v1.24.md

```diff
@@ -4823,9 +4823,9 @@ Return information on the service `id`.
 
 #### Update a service
 
-`POST /services/(id or name)/update`
+`POST /services/(id)/update`
 
-Update a service. When using this endpoint to create a service using a
+Update a service with full ID. When using this endpoint to create a service using a
```

Comment:
But we don't specify that a prefix ID can be used anywhere in this document. Without mentioning that, a full ID should be assumed here.